sdk/schemaregistry/schema-registry/src/schemaRegistryClient.ts

```diff
@@ -78,18 +88,20 @@ export class SchemaRegistryClient implements SchemaRegistry {
    */
   async registerSchema(
     schema: SchemaDescription,
-    options?: RegisterSchemaOptions
+    options: RegisterSchemaOptions = {}
   ): Promise<SchemaProperties> {
     const { groupName, name: schemaName, definition: schemaContent, format } = schema;
-    return this.client.schema
-      .register(
-        groupName,
-        schemaName,
-        `application/json; serialization=${format}`,
-        schemaContent,
-        options
-      )
-      .then(convertSchemaIdResponse(format));
+    return this._tracing.withSpan("registerSchema", options, (updatedOptions) =>
```

Comment:
I think the convention we're landing on is `ClassName.functionName` here - so in this case I think SchemaRegistryClient.registerSchema is the name of the span